install-nvm.sh

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+
+wget https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh
+php -r '
+    $hash = "dd4b116a7452fc3bb8c0e410ceac27e19b0ba0f900fe2f91818a95c12e92130fdfb8170fec170b9fb006d316f6386f2b";
+    if (hash_file("sha384", "install.sh") === $hash) {
+    echo "Installer verified";
```

Comment:
indentation

Reply:
Please don't just say "indentation". If there's a specific problem, please either use the code change suggestion tool to indicate what specific change you want, or use full sentences to tell me what needs to change.
I shouldn't have to re-review my code to see if I can decipher what you mean.

Reply:
Done